### PR TITLE
EPMRPP-38516: part of EPMRPP-33317 Add form validation for add/edit defect form

### PR DESCRIPTION
**Pull request by margalencia**
+ small refactoring

image

### CODE REVIEW

File: app/localization/translated/be.json [hunk 1]
```diff
@@ -971,16 +985,15 @@
...
   "NotificationCase.attributesLabel": "Атрыбуты (і)",
   "NotificationCase.attributesNote": "Дачылаць апавяшчэнні пра запускі, якія ўтрымліваюць пазначаныя атрыбуты",
   "NotificationCase.controlPanelName": "Правіла",
-  "NotificationCase.always": "Заўсёды",
-  "NotificationCase.failed": "Запуск утрымлівае праблемы",
-  "NotificationCase.more10": "> 10% складнікаў утрымліваюць праблемы",
-  "NotificationCase.more20": "> 20% складнікаў утрымліваюць праблемы",
-  "NotificationCase.more50": "> 50% складнікаў утрымліваюць праблемы",
-  "NotificationCase.toInvestigate": "Запуск утрымлівае складнікі, што патрабуюць аналізу",
+  "NotificationCase.dropdownValueAlways": "Always",
```
Comment (DiscoElevator) [inline app/localization/translated/be.json]: I thinks it's better to use here translated strings from above

File: app/localization/translated/ru.json [hunk 2]
```diff
@@ -971,16 +985,15 @@
...
   "NotificationCase.attributesLabel": "Атрибуты (и)",
   "NotificationCase.attributesNote": "Отправлять уведомления о запусках, содержащих обозначенные атрибуты",
   "NotificationCase.controlPanelName": "Правило",
-  "NotificationCase.always": "Всегда",
-  "NotificationCase.failed": "Запуск содержит ошибки",
-  "NotificationCase.more10": "> 10% элементов содержат ошибки",
-  "NotificationCase.more20": "> 20% элементов содержат ошибки",
-  "NotificationCase.more50": "> 50% элементов содержат ошибки",
-  "NotificationCase.toInvestigate": "Запуск имеет элементы требующие анализа",
+  "NotificationCase.dropdownValueAlways": "Always",
```
Comment (DiscoElevator) [inline app/localization/translated/ru.json]: The same as the comment above

File: app/localization/translated/be.json (hunk 1 above)
Comment (margalencia) [reply app/localization/translated/be.json]: Actually, `NotificationCase` strings are a part of someone else changes, but I've updated the translations

File: app/localization/translated/ru.json (hunk 2 above)
Comment (margalencia) [reply app/localization/translated/ru.json]: done

**Pull request merged**